### PR TITLE
Add native version, fix license headers

### PR DESCRIPTION
**Pull request by bibikar**
- Fix license headers to also have MIT license
- Add native versions and Makefile
- Update readme

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Copyright (C) 2014-2015 Intel Corporation
```
Comment (anton-malakhov) [inline GNUmakefile]: `, 2018` ?

File: GNUmakefile [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+# ==============================================================================
+
+all: bin
+	./$(TARGET)
+
+bin: $(TARGET)
+
+
+$(TARGET): $(SRC)
+	$(CC) $(CFLAGS) $(SRC) -mkl -o $(TARGET)
```
Comment (anton-malakhov) [inline GNUmakefile]: add `$(CPPFLAGS)` for preprocessor?

File: README.md [hunk 3]
```diff
@@ -1,15 +1,29 @@
 # BlackScholes benchmark
 Benchmark computing Black Scholes formula using different technologies
 
+## Prerequisites
+- `icc`, if compiling native benchmarks. Intel Distribution for Python*
+  2019 Gold benchmarks used icc 17.0.1.
+- `mkl`, if compiling native benchmarks.
```
Comment (anton-malakhov) [inline README.md]: mkl should really be optional if you want just SVML benches

File: base_bs_erf.py [hunk 4]
```diff
@@ -1,27 +1,6 @@
-# Copyright (c) 2017, Intel Corporation
+# Copyright (C) 2017 Intel Corporation
```
Comment (anton-malakhov) [inline base_bs_erf.py]: `-2018`? here and everywhere

**Review by anton-malakhov — APPROVED** [review]
LGTM, few improvements suggested

File: GNUmakefile [hunk 5]
```diff
@@ -43,6 +48,11 @@ CFLAGS      += -qopenmp
 CFLAGS      += -I./
 CFLAGS		+= -fp-model precise
 
+ifeq ($(MKL),1)
+	CFLAGS += -mkl
+	CFLAGS += -DMKL
```
Comment (anton-malakhov) [inline GNUmakefile]: the compiler flag should be more unique in order to avoid collision with MKL itself

File: GNUmakefile [hunk 6]
```diff
@@ -22,10 +22,15 @@
...
-       black-scholes_mkl.c \
        data_gen.c          \
        main.c
 
+MKL ?= 1
+
+ifeq ($(MKL),1)
+	SRC += black-scholes_mkl.c
+endif
+
```
Comment (anton-malakhov) [inline GNUmakefile:27]: BTW, why not to create a separate target instead?

**Review by anton-malakhov — CHANGES_REQUESTED** [review]

File: GNUmakefile (hunk 6 above)
Comment (bibikar) [reply GNUmakefile:27]: done!

**Review by anton-malakhov — APPROVED** [review]
thanks!

**Pull request merged**